content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -216,70 +203,124 @@ Registry servers configured in HTTP mode require the `--insecure` CLI flag when
 spectro registry login --insecure http://example.com:5000
 ```
 
-
 </WarningBox>
 
 </Tabs.TabPane>
 
-
 </Tabs> 
-
-    
+ 
 <br />
 
 
-
 9. Expose the container host's port publicly to allow the console to interact with the pack registry.
    This would be typically done via environment-specific constructs like Security Groups, Firewalls, etc.
 
 
 10. Verify the installation by invoking the pack registry APIs using the curl command. This should result in a 200 response.
 
-  * **HTTPS mode**
+<Tabs>
 
-    ```bash
-    curl --cacert tls.crt -v [REGISTRY_SERVER]/health
-    curl --cacert tls.crt -v -u [USERNAME] [REGISTRY_SERVER]/v1/_catalog
-    ```
+<Tabs.TabPane tab="HTTPS" key="https-2">
 
-  * **HTTP mode**
+  ```bash
+  curl --cacert tls.crt -v [REGISTRY_SERVER]/health
+  curl --cacert tls.crt -v -u [USERNAME] [REGISTRY_SERVER]/v1/_catalog
+  ```
 
-    ```bash
-    curl -v [REGISTRY_SERVER]/health
-    curl -v -u [USERNAME] [REGISTRY_SERVER]/v1/_catalog
-    ```
+</Tabs.TabPane>
 
-# Configure a Custom Pack Registry on the Palette Console
+<Tabs.TabPane tab="HTTP" key="http-2">
 
-Once the deployment of the pack registry server is complete, configure it with the console as follows:
+  ```bash
+  curl -v [REGISTRY_SERVER]/health
+  curl -v -u [USERNAME] [REGISTRY_SERVER]/v1/_catalog
+  ```
+
+</Tabs.TabPane>
+
+</Tabs> 
 
-1. As a Tenant Administrator, navigate to **Admin Settings** > **Registries** > **Pack Registries**.
 
 
-2. Click on **Add New Pack Registry** and provide the pack registry name, endpoint, and user credentials.
+# Configure a Custom Pack Registry in Palette
 
+<<<<<<< HEAD
+Once you deploy the pack registry server, use the following steps to configure the pack registry server in Palette.
+<br />
+
+1. Log in to Palette, and switch to the tenant admin view. 
+
+=======
+# Configure a Custom Pack Registry in Palette
+
+Once you deploy the pack registry server, use the following steps to configure the pack registry server in Palette.
+<br />
 
-3. Click on **Confirm** once the details are filled.
+1. Log in to Palette, and switch to the Tenant admin view. 
```

Comment:
```suggestion
1. Log in to Palette, and switch to the tenant admin view. 
```
We agreed in a team meeting to use all lowercase for tenant admin - [Approved Word list](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933078/Word+List).